doc/api/stream.md

```diff
@@ -937,7 +937,7 @@ the internal buffer is fully drained.
 const readable = getReadableStreamSomehow();
 readable.on('readable', () => {
   let chunk;
-  while (null !== (chunk = readable.read())) {
+  while ((chunk = readable.read()) !== null) {
```

Comment:
Type-check-on-RHS is less readable than the other way around in expressions like these, IMO.

Reply:
I think this is something very subjective. I personally feel like the version is better to read.